state-chain/chains/src/sol/api.rs

```diff
@@ -383,13 +383,20 @@ impl<Environment: SolanaEnvironment> SolanaApi<Environment> {
 		ccm_additional_data: VersionedSolanaCcmAdditionalData,
 	) -> Result<Self, SolanaTransactionBuildingError> {
 		let ccm_accounts = ccm_additional_data.ccm_accounts();
+		let alts = ccm_additional_data
+			.address_lookup_tables()
```

Comment:
I think renaming this to be something like `address_lookup_table_addresses` makes it a bit clearer, since here they are just the addresses of the tables, rather than the tables themselves.